src/Cohesity.Powershell/Scripts/Clusters/Get-CohesityWhiteListIP.ps1

```diff
@@ -0,0 +1,43 @@
+function Get-CohesityWhiteListIP {
+    <#
+        .SYNOPSIS
+        Get whitelist IPs.
+        .DESCRIPTION
+        The Get-CohesityWhiteListIP function is used to get whitelist IP(s).
+        .NOTES
+        Published by Cohesity
+        .LINK
+        https://cohesity.github.io/cohesity-powershell-module/#/README
+        .EXAMPLE
+        Get-CohesityWhiteListIP
+    #>
+    [CmdletBinding()]
+    Param(
+    )
+
+    Begin {
+        if (-not (Test-Path -Path "$HOME/.cohesity")) {
+            throw "Failed to authenticate. Please connect to the Cohesity Cluster using 'Connect-CohesityCluster'"
+        }
+        $cohesitySession = Get-Content -Path $HOME/.cohesity | ConvertFrom-Json
```

Comment:
Have we not started to use the session variable instead of storing the credentials in a file ?

Reply:
We are not storing the credentials, instead here we are using the session token stored in the file.

Reply:
right, we had agreed to remove this approach and make it take it from env variable no ?